src/AWS.Deploy.Common/Recipes/PropertyDependency.cs

```diff
@@ -9,14 +9,22 @@ namespace AWS.Deploy.Common.Recipes
     public class PropertyDependency
     {
         public string Id { get; set; }
+        public PropertyDependencyOperationType? Operation { get; set; }
         public object Value { get; set; }
 
         public PropertyDependency(
             string id,
+            PropertyDependencyOperationType? operation,
             object value)
         {
             Id = id;
+            Operation = operation;
             Value = value;
         }
     }
+
+    public enum PropertyDependencyOperationType
+    {
+        NotEmpty
```

Comment:
Can we add `Equals` in this enum as well which is the implicit default.

Reply:
Done